packages/admin/app/routes/peers.tsx

```diff
@@ -0,0 +1,10 @@
+import { Outlet } from '@remix-run/react'
+import styles from '../styles/dist/main.css'
+
+export default function PeersBasePage() {
+  return <Outlet />
```

Comment:
You probably don't need this complexity.

You could replace the contents of this file with the contents of `/peers/index.tsx` and then remove the directory, and then use the [dot delimeters](https://remix.run/docs/en/v1/file-conventions/routes-files#dot-delimiters) for child pages. Because these pages aren't really nested. So `/peers/create.tsx` just becomes `/peers.create.tsx`